migrating/ossm-migrating-from-2-to-3-assembly.adoc

```diff
@@ -6,4 +6,80 @@ include::_attributes/common-attributes.adoc[]
+====
+
+//02/04/2024: Per Service Mesh Program Call, there will be a 2.6.6 release with 3.0, so users will need to upgrade to 2.6.6. UPDATE THE VARIABLE IN COMMON_ATTRIBUTES ACCORDINGLY
+//it looks like the latest z-stream will be 2.6.5, set to release on Feb 5, 2025. Unless there is a z-stream in Feb, 2.6.5 is likely the version to be listed for 2.6.z.
+
+[id="recommendations_{context}"]
+== Recommendations
+
+Given the nature of migrating to a brand new operator, the following are recommendations to help limit the risk of misconfigurations or possible conflicts:
+
+* If you have automated updates configured in the {ocp-product-title} web console, change it to **Manual**. This helps to prevent updates to the {SMProduct} 2 Operator and control planes during your migration.
+
+* Keep the amount of the service mesh configuration changes, such as changes for traffic management or security or adding new workloads or gateways, to a minimum during the migration.
+
+[NOTE]
+====
+If it's necessary to add a new workload namespace during the migration, it must be managed by the {SMProduct} 3 control plane and labeled with `maistra.io/ignore-namespace: "true"` to avoid conflicts between the {SMProduct} 3 control plane and the {SMProduct} 2 `ServiceMeshControlPlane` resource.
+====
+
+* Finish the migration without unnecessary delays
+
+Once the migration is started, it is recommended to complete it as quickly as possible.
+
+[id="ow-to-use-migrating-guides_{context}"]
+== How to use the migration guides
+
+The migration guides are designed to help you move from {SMProduct} {SMv2Version} to {SMProduct} 3.0, based on your deployment model:
+
+* Mulitenant
+* Mulitenant with cert-manager
+* Cluster-wide
+* Cluster-wide with cert-manager
+
+The migration guides use the `bookinfo` application as an example for demonstrations purposes.
+
+You can also migrate gateways.
+
+[NOTE]
+====
+From time to time, you may need to reference, or will be directed to {SMProduct} 2.x content. It may be benefial to open link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x[OpenShift Service Mesh 2.x] in a new tab or window for easier reference. This may be especially helpful when moving between {SMProduct} 2 `ServiceMeshControlPlane` resource content and {SMProduct} 3 `Istio` resource content.
+====
+
+[id="premigration-checklists{context}"]
+=== Premigration checklists
+
+There are a series of premigration checklists that must be completed before you can begin migrating your deployment and workloads.
+
+These checklists include steps for additional configurations if you must recreate your network policies before you migrate your deployment and workloads, and configuration updates to set up add ons such as {kialiproduct}, and {temposhortname}.
```

Comment:
Whether or not they will recreate network policies will depend on whether or not they will use them so maybe we can just keep this short?
```suggestion
These checklists include steps for managing your network policies and configuration updates to set up add ons such as {kialiproduct}, and {temposhortname}.
```